data_curation_app.py

```diff
@@ -1,49 +1,49 @@
-parser.add_argument("-o", "--output", dest="output", help="path of output files", required=True)
-args = parser.parse_args()
-OCR_results_path = args.input
-images_path = args.image
-output_path = args.output
+from utils import resize_image, scale_bounding_box, handle_wrong_datapoint, handle_missing_datapoint, handle_user_choice
+import hosting_config
 
 
 def next_page():
     if len(st.session_state["OCR_output_files"]) > 0:
         current_file = st.session_state["OCR_output_files"].pop(0)
         st.session_state["curation_output_files"].append(current_file)
-        shutil.move(os.path.join(st.session_state["label_folder_path"], current_file), os.path.join(st.session_state["output_folder_path"], current_file))
+        source_key = os.path.join(st.session_state["label_folder_path"], current_file)
+        destination_key = os.path.join(st.session_state["output_folder_path"], current_file)
+        hosting_config.file_manager.move_files(source_key=source_key, destination_key=destination_key)
         st.session_state["previous_clicked"] = False
-
     st.experimental_rerun()
 
 
 def previous_page():
     current_file = st.session_state["curation_output_files"].pop()
     st.session_state["OCR_output_files"].insert(0, current_file)
-    shutil.move(os.path.join(st.session_state["output_folder_path"], current_file), os.path.join(st.session_state["label_folder_path"], current_file))
+    source_key = os.path.join(st.session_state["output_folder_path"], current_file)
+    destination_key = os.path.join(st.session_state["label_folder_path"], current_file)
+    hosting_config.file_manager.move_files(source_key=source_key, destination_key=destination_key)
     st.session_state["previous_clicked"] = True
-
     st.experimental_rerun()
 
-
-def handle_image_and_bounding_box(OCR_results_file, images_path, bounding_boxes):
-    image_file_name = get_image_file_name(OCR_results_file, images_path)
+@st.cache_data()
+def handle_image_and_bounding_box(OCR_results_file, bounding_boxes):
+    image_file_name = hosting_config.file_manager.get_image_file_name(OCR_results_file)
     if image_file_name is None:
         next_page()
-    image = read_document(image_file_name, images_path)
-    image, width_scale, height_scale = resize_image(image, 800, 800)
+    images = hosting_config.file_manager.read_document(image_file_name)
+    resized_images = []
+    scaled_bounding_boxes = []
+
+    for i, img in enumerate(images):
+        img, width_scale, height_scale = resize_image(img, 800, 800)
+        resized_images.append(img)
 
-    bounding_boxes = scale_bounding_box(bounding_boxes, width_scale, height_scale)
-    return image, image_file_name, bounding_boxes
+        page_scaled_bounding_boxes = scale_bounding_box(bounding_boxes, width_scale, height_scale, i + 1)
+        scaled_bounding_boxes.extend(page_scaled_bounding_boxes)
+
+    bounding_boxes["objects"] = scaled_bounding_boxes
```

Comment:
How can we still be changing this at this point ? I don't understand. This makes my review very difficult